src/sentry/feedback/endpoints/project_feedback_details.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework.request import Request
```

Comment:
we can just remove this file from the PR as well and keep it only ingestion for now -- can follow up with the feedback details endpoint in a separate PR